src/logistration/LoginPage.jsx

```diff
@@ -48,14 +49,22 @@ class LoginPage extends React.Component {
     };
   }
 
+  componentDidMount() {
+    const params = (new URL(document.location)).searchParams;
+    const payload = {
+      redirect_to: params.get('next') || DEFAULT_REDIRECT_URL,
+    };
+    this.props.getThirdPartyAuthContext(payload);
+  }
+
   handleSubmit = (e) => {
     e.preventDefault();
-    const params = getQueryParameters();
+    const params = (new URL(document.location)).searchParams;
```

Comment:
just for clarification why we are removing `getQueryParameters`

Reply:
This is deprecated now in favour of `getQueryParameters()`